### PR TITLE
Added get attribute data function for ngsiv2

### PR DESCRIPTION
**Pull request by manucarrace**
- Added nue funciotn for get attribute data operation in NGSIv2

### CODE REVIEW

File: iotqatools/cb_ngsiv2_utils.py [hunk 1]
```diff
@@ -364,7 +408,7 @@ def list_entities(self, headers, filters=None):
     print(attr.get_attribute())
```
Comment (b-rvc) [inline iotqatools/cb_ngsiv2_utils.py:408]: Use logging instead of print

**Comment by XavierVal** [comment]
LGTM

**Comment by jcanonav** [comment]
LGTM

**Comment by XavierVal** [comment]
Please increase a minor setup version prior to merge

**Comment by iariasleon** [comment]
LGTM

File: iotqatools/cb_ngsiv2_utils.py (hunk 1 above)
Comment (manucarrace) [inline iotqatools/cb_ngsiv2_utils.py:408]: Print, only for debugging purposes in test. Theres no logger defined in the lib

**Comment by manucarrace** [comment]
Version increased in b2f5d2628482e5d2cffef4cc3826c53f7a4153e3

**Pull request merged**